.github/DISCUSSION_TEMPLATE/css-battles.yml

```diff
@@ -0,0 +1,48 @@
+title: "CSS Battle #NUMBER – NAME"
+labels: ["CSS Battles"]
+body:
+  - type: input
+    id: link
+    attributes:
+      label: "Link to battle:"
+      description: Replace NUMBER with current battle number      
+      value: |
+       [Lets battle! ⚔️](https://cssbattle.dev/play/NUMBER)
+    validations:
+      required: true
+  - type: markdown
+    attributes: 
+      value: |
+        **That’s all! You can now click the “Start discussion” button.** :smile:
+        **Please note:** Don't change text areas - they need to stay that way to be visible after creating discussion.
+  - type: textarea
+    attributes:
+      label:  "Copy the code block below to format your comment on the discussion thread:"
+      description: Solution template for players
+      value: |
+        ```html
+        <details>
+        <summary>Description – <strong>score {characters}</strong></summary><br />
+        <!-- Has to be followed by an empty line! -->
+
+        <!-- remove + character -->
+        +```html
+        Your CSSBattles.dev code here! ⚔️
+        +```
+
+        </details>
+        ```

```

Comment:
We can fix up the formatting here as well (getting rid of the "remove + character" step):

```suggestion
        ````md
        <details>
        <summary>Code Source  – <strong>score {character count}</strong></summary>
        <!-- have to be followed by an empty line! -->

        ```html
        your cssbattles code here!
        ```
        </details>
        ````

```

Reply:
Wow! I didn't know that! This is **_extremely_** helpful and what we were trying to do all along. Thanks, @danieltott!